FoodTracker/RatingControl.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  RatingControl.swift
+//  FoodTracker
+//
+//  Created by Josip Glasovac on 10/10/2018.
+//  Copyright © 2018 Josip Glasovac. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable class RatingControl: UIStackView {
+
+    @IBInspectable var starts: Int = 5 {
+        didSet {
+            setupButtons()
+        }
+    }
+
+    @IBInspectable var starSize: CGSize = CGSize(width: 44.0, height: 44.0) {
+        didSet {
+            setupButtons()
+        }
+    }
+
+    var rating = 0 {
+        didSet {
+            updateButtonSelectionStates()
+        }
+    }
+
+    private var ratingButtons = [UIButton]()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupButtons()
+    }
+
+    required init(coder: NSCoder) {
+        super.init(coder: coder)
+
+        setupButtons()
+    }
+
+    private func setupButtons() {
+
+        clearButtons()
+
+        let bundle = Bundle(for: type(of: self))
+        let filledStar = UIImage(named: "filledStar", in: bundle, compatibleWith: self.traitCollection)
```

Comment:
Because of @IBDesignable, It's need to init Bundle if you want to see buttons that is defined in code on Interface Builder. But I don't need it anymore so I don't need Bundle init. 

"because the control is @IBDesignable, the setup code also needs to run in Interface Builder. For the images to load properly in Interface Builder, you must explicitly specify the catalog’s bundle. This ensures that the system can find and load the image."